Torchaudio build failure

In E2E and nightly wheels workflows.

https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10642694426/job/29505435952
https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10641254851

```
CMake Error at cmake/TorchAudioHelper.cmake:18 (add_library):
  Target "libtorchaudio" links to target "torch::xpurt" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```

--- a/.github/workflows/nightly-wheels.yml
+++ b/.github/workflows/nightly-wheels.yml
@@ -47,13 +47,6 @@ jobs:
         with:
           repository: pytorch/pytorch
 
-      - name: Identify pinned versions
-        run: |
-          cd pytorch
-          echo "TORCHVISION_COMMIT_ID=$(<.github/ci_commit_pins/vision.txt)" >> $GITHUB_ENV
-          echo "TORCHTEXT_COMMIT_ID=$(<.github/ci_commit_pins/text.txt)" >> $GITHUB_ENV
-          echo "TORCHAUDIO_COMMIT_ID=$(<.github/ci_commit_pins/audio.txt)" >> $GITHUB_ENV
-
       - name: Identify Triton commit id
         run: |
           echo "TRITON_COMMIT_ID=$(git rev-parse HEAD)" >> $GITHUB_ENV
@@ -87,38 +80,11 @@ jobs:
           path: ${{ steps.triton-cache.outputs.path }}
           dest: ${{ steps.triton-cache.outputs.dest }}
 
-      - name: Install torchvision package
-        uses: ./.github/actions/install-dependency
-        with:
-          package: torchvision
-          repository: pytorch/vision
-          ref: ${{ env.TORCHVISION_COMMIT_ID }}
-          extra-cache-key: ${{ env.PYTORCH_VERSION }}
-
-      - name: Install torchtext package
-        uses: ./.github/actions/install-dependency
-        with:
-          package: torchtext
-          repository: pytorch/text
-          ref: ${{ env.TORCHTEXT_COMMIT_ID }}
-          extra-cache-key: ${{ env.PYTORCH_VERSION }}
-
-      - name: Install torchaudio package
-        uses: ./.github/actions/install-dependency
-        with:
-          package: torchaudio
-          repository: pytorch/audio
-          ref: ${{ env.TORCHAUDIO_COMMIT_ID }}
-          extra-cache-key: ${{ env.PYTORCH_VERSION }}
-
       - name: Prepare wheels for upload
         run: |
           mkdir -p wheels
           cp -L pytorch/dist/*.whl wheels/
           cp -L python/dist/*.whl wheels/
-          cp -L torchvision*/dist/*.whl wheels/
-          cp -L torchtext*/dist/*.whl wheels/
-          cp -L torchaudio*/dist/*.whl wheels/
 
       - name: Report environment details
         run: |
@@ -137,9 +103,6 @@ jobs:
           LLVM_COMMIT_ID=$LLVM_COMMIT_ID
           TRITON_REPO=intel/intel-xpu-backend-for-triton
           TRITON_COMMIT_ID=$TRITON_COMMIT_ID
-          TORCHVISION_COMMIT_ID=$TORCHVISION_COMMIT_ID
-          TORCHTEXT_COMMIT_ID=$TORCHTEXT_COMMIT_ID
-          TORCHAUDIO_COMMIT_ID=$TORCHAUDIO_COMMIT_ID
           EOF
 
       - name: Upload PyTorch wheels to artifacts

